src/css/home.css

```
@@ -114,3 +127,62 @@ footer a {
 footer .sns {
 	gap: 12px;
 }
+
+/* responsive */
+@media (max-width: 1199px) {
+	.wrapper > section.inner {
+		/* max-width: 696px; */
+		padding-left: 24px;
+		padding-right: 24px;
+		padding: 24px;
```

```suggestion
		padding: 24px;
```
한 번만 작성해주셔도 될 것 같습니다.